src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/GeneralPartnerServiceTest.java

```diff
@@ -21,7 +21,9 @@
 import uk.gov.companieshouse.limitedpartnershipsapi.repository.GeneralPartnerRepository;
 import uk.gov.companieshouse.limitedpartnershipsapi.utils.TransactionUtils;
 
+import java.util.ArrayList;
```

Comment:
Looks like you've removed a test from a previous commit but left unused imports and variables.

Reply:
Tidied this up.

Reply:
Yep, but as there are no changes to the file (other than a blank line that's been deleted), just revert it so it's not part of the PR.

Reply:
Removed line change.